Doc/tutorial/modules.rst

```diff
@@ -167,7 +167,7 @@ executed as the "main" file:
    $ python fibo.py 50
    0 1 1 2 3 5 8 13 21 34
 
-If the module is imported, the code is not run::
+If the module is imported, then the code will not run::
```

Comment:
There is no error here!
«then» is implicit here in the juxtaposition, and «is not run» is an accurate description of what happens.